sys/include/net/loramac.h

```diff
@@ -203,14 +210,22 @@ extern "C" {
  * @brief   Default second RX window datarate index
  */
 #ifndef LORAMAC_DEFAULT_RX2_DR
+#ifdef LORAMAC_REGION_EU_868
 #define LORAMAC_DEFAULT_RX2_DR                 (LORAMAC_DR_0)
+#elif defined(LORAMAC_REGION_IN_865)
```

Comment:
Missing IS_ACTIVE here.

Also as described in another comment, this should be at least prefixed with "CONFIG_"